web/modules/custom/custom_page/src/Plugin/Block/MenuSubPages.php

```diff
@@ -0,0 +1,242 @@
+    parent::__construct($configuration, $plugin_id, $plugin_definition);
+    $this->entityTypeManager = $entity_type_manager;
+    $this->entityRepository = $entity_repository;
+    $this->joinupCoreRelationsManager = $relation_manager;
+    $this->menuLinkManager = $menu_link_manager;
+    $this->currentRouteMatch = $route_match;
+    $this->membershipManager = $membership_manager;
+
+    $collection_contexts = $collection_context->getRuntimeContexts(['og']);
+    if ($collection_contexts && $collection_contexts['og']->hasContextValue()) {
+      $this->collection = $collection_contexts['og']->getContextValue();
+      $results = $this->membershipManager->getGroupContentIds($this->collection, ['ogmenu_instance']);
+
+      if (!empty($results)) {
+        $this->ogMenuInstanceId = reset($results['ogmenu_instance']);
+      }
+    }
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function create(ContainerInterface $container, array $configuration, $plugin_id, $plugin_definition) {
+    return new static(
+      $configuration,
+      $plugin_id,
+      $plugin_definition,
+      $container->get('entity_type.manager'),
+      $container->get('entity.repository'),
+      $container->get('joinup_core.relations_manager'),
+      $container->get('collection.collection_route_context'),
+      $container->get('plugin.manager.menu.link'),
+      $container->get('current_route_match'),
+      $container->get('og.membership_manager')
+    );
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function build() {
+    $build = [];
+    $items = [];
+    $child_links = $this->getChildLinks();
+    if (empty($child_links)) {
+      return $build;
+    }
+    foreach ($child_links as $link) {
+      $parameters = $link->getUrlObject()->getRouteParameters();
+      $node_id = $parameters['node'];
+      $custom_page = $this->entityTypeManager->getStorage('node')->load($node_id);
+      $build = $this->entityTypeManager->getViewBuilder('node')->view($custom_page, 'view_mode_tile');
+      $items[$link->getWeight()] = [
+        '#type' => 'container',
+        '#weight' => $link->getWeight(),
+        '#attributes' => [
+          'class' => [
+            'listing__item',
+            'listing__item--tile',
+            'mdl-cell',
+            'mdl-cell--4-col',
+          ],
+        ],
+        $custom_page->id() => $build,
+      ];
+    }
+
+    $build = [
+      '#type' => 'item',
+      0 => [
+        '#type' => 'item',
+        '#wrapper_attributes' => [
+          'class' => ['listing', 'listing--grid', 'mdl-grid'],
+        ],
+      ],
+    ];
+    $build[0] += $items;
```

Comment:
The `item` type is mostly meant for forms, `container` should be used.
```
$build = [
  // The 'listing' child key is needed to avoid copying the #attributes to
  // the parent block.
  // @see \Drupal\block\BlockViewBuilder::preRender()
  'listing' => [
    '#type' => 'container',
    '#attributes' => [
      'class' => ['listing', 'listing--grid', 'mdl-grid'],
    ],
  ],
];
$build['listing'] += $items;
```